l10n_it_ricevute_bancarie/models/account.py

```diff
@@ -191,16 +191,12 @@ def button_draft(self):
         # ---- line was added on new validate
         super(AccountMove, self).button_draft()
         for invoice in self:
-            invoice.write(
-                {
-                    "invoice_line_ids": [
-                        (2, line.id, 0)
-                        for line in invoice.invoice_line_ids
-                        if line.due_cost_line
-                    ]
-                }
-            )
-            invoice._recompute_tax_lines()
+            line_ids_to_remove = self.get_due_cost_line_ids(invoice)
```

Comment:
In base alla tua ultima modifica https://github.com/OCA/l10n-italy/compare/763e05edd0787fca13bf8029d53230ad7e857fa8..a4602c65e1a85c6a5a840e6839171c6d8cb36a36 modificherei il nome di questa variabile in `due_cost_line_ids` in modo da renderlo coerente con ciò che restituisce il metodo. Che ne pensi?

Reply:
Si può andar bene! Lo avevo chiamato così per esser chiari sull'utilizzo che se ne fa successivamente
Ora lo cambio

Reply:
:+1: